### ISSUE TITLE
Possible bugs For getting the logits value

### ISSUE THREAD

**Issue by yunyiliu** [sathishkumar67/Simpo#1]
# get logits for chosen and rejected sequences
        chosen_ids_logits = self.model(chosen_ids, attention_mask=chosen_attention_mask)
        rejected_ids_logits = self.model(rejected_ids, attention_mask=rejected_attention_mask)
For the above code, I believe it should be like the following?

        chosen_ids_logits = self.model(chosen_ids, attention_mask=chosen_attention_mask).logits
        rejected_ids_logits = self.model(rejected_ids, attention_mask=rejected_attention_mask).logits

Because the model not only return the logits back?
